exercises/markdown/markdown_test.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+from markdown import parse_markdown
+
+
```

Comment:
Could you please also leave a comment stating what version of `canonical-data.json` the tests were adopted as discussed in #784?

Reply:
@PaulGlass, the new format is:
`# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.0`

Reply:
Fixed